README.md

```diff
@@ -1,77 +1,70 @@
-  
-  ## How to complete homework? 
-  ### Implementation of homework
-  For the first chapters, homework branch will come with 2 projects:
-  ```
-  ChapterName.csproj
-  ChapterName.Tests.csproj
-  ```
-  For now, a student should not care about about the tests project, as long as it builds. 
-  All the student work will happen in **ChapterName.csproj**. 
-  You will have empty functions which you will have to implement based on the homework requirements. 
-  Student can consider a **homework complete if all tests pass**.
-  
-  ### Homework submission
- 
-  In order for mentors to review your homework, you will need to create a Pull Request (PR) 
-  How to do it can be found here: (TODO: add link) 
-  
-  ## Homework review
-  ### Automatic code reviewer
-  To automate standards review, we have a static analysis tool--Codacy. It will grade your code (A- being the highest grade). Homework will not be considered complete, if grade is D or lower. 
-  ### Auto build and auto tests
-  In order to find out if your code works not just on your machine but in all environments, check the build status. 
-  In PR you will see Azure Pipelines and it should have x check passed. 
-  Todo: screen shot
-  ### Mentor code review
-  In order for your PR to be complete, it needs a code review of at least 2 mentors. 
-  Both of the mentors will have to approve your code, otherwise you won't have a PR completed. 
-  How to do code review:
-  As a reviewer- Todo: link
-  As a reviewee-Todo: link
-  Code reviews on GitHub- Todo: link
-  
-  ### Finishing homework
-  You can consider your homework complete if all of the above steps pass. 
-  **Don't merge the PR to master branch**
-  A single best homework implementation will be merged to master. 
-
-  
+# Homework 3
 
-  # Community
-  If you want to join as either mentor or pupil or have any questions, please refer to our discord Community.  
-  https://discord.gg/rCMKcUU
-  
-  
-
-
-
- 
-
- 
+## Hints
+* This will make homework 4 much more easy:
+  * Do use Environment.NewLine if you want to concanate a string with a newline command (or use WriteLine).
+  * Do use [CultureInfo.InvariantCulture](https://docs.microsoft.com/de-de/dotnet/api/system.globalization.cultureinfo.invariantculture?view=netcore-3.1) when reading in float numbers. (Due to differences in parsing numbers with '.' and ',' character.
+* The better your design, the easier it will be to find and fix errors!
+* Sometimes you have to sync new changes from the C# Inn repository:
+  * Click the pencil symbol at the bottom right.
+  * Commit your changes or stash them (after finishing homework 2 there should none be left).
+  * Choose the correct branch in the bottom right (in this case Chapter1/Homework/3 should already be selected).
+  * Click the house button straight above (upper end).
+  * Click the sync button.
+  * Click fetch.
+  * Click pull.
 
+## Task
+1) Open the Homework 3:
+  * Choose the Chapter1/Homework/3 branch in the bottom right.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[list-item-bullet-indent] Incorrect indentation before bullet: remove 2 spaces](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4896222)